ci/scripts/audit.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+export cwd=$(pwd)
+
+pushd $cwd/dp-developer-site
```

Comment:
This path is correct, but you will need to update this line in the `build.sh` to match after moving to go modules.

Reply:
Updated